index.rst

```diff
@@ -74,15 +119,16 @@ The manual can be found on Docushare `here <https://docushare.lsst.org/docushare
 
 Fiber Spectrograph
 ------------------
-The fiber spectrograph used is an Avantes SensLine AvaSpecULS2048x64TEC-EVO, with a wavelength range of 200-1160 nm. An optical fiber runs from the fiber spectrograph and samples the light from the optical path after it exits from the monochromator. 
+The fiber spectrograph used is an Avantes SensLine AvaSpecULS2048x64TEC-EVO, with a wavelength range of 200-1160 nm. An optical fiber runs from the fiber spectrograph and samples the light from the optical path after it exits the monochromator. 
 
-The fiber spectrograph is controlled via USB that runs directly from the electrometer to a linux control computer (loonie) in the electronics cabinet. It can be commanded via CSC. 
+The fiber spectrograph is controlled via USB that runs directly from the fiber spectrograph to a linux control computer (loonie) in the electronics cabinet. It can be commanded via CSC. 
+
+We also have a fiber illuminator (Spectral Cal Lamp), which can be used to calibrate the Fiber Spectrograph. Power is avaialble for the illuminator on the table, but we expect that the illuminator will only be installed when needed. When we want to calibrate the fiber spectrograph, simply move the fiber from the optical path and install it into the illuminator.
 
-We also have a fiber illuminator, which can be used to calibrate the Fiber Spectrograph. Power is avaialble for the illuminator on the table, but we expect that the illuminator will only be installed when needed. When we want to calibrate the fiber spectrograph, simply move the fiber from the optical path and install it into the illuminator.
 
 Electrometer
```

Comment:
With each of these hardware components, it'd been good to add one sentence at the end that says something like:
"The Electrometer is commanded by the `ts_electrometer` CSC, which has it's configuration stored in `ts_config_ocs``

Reply:
Added this general sentence to the other components as well.